functional/security/Crypto/playlist.xml

```diff
@@ -17,9 +17,9 @@
 		<testCaseName>CryptoTests</testCaseName>
 		<disables>
 			<disable>
-				<comment>https://github.com/eclipse-openj9/openj9/issues/16712</comment>
-				<version>19+</version>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/18801</comment>
 				<impl>openj9</impl>
+				<testflag>fips140_2</testflag>
```

Comment:
I wonder if it should be like https://github.com/eclipse-openj9/openj9/blob/master/test/functional/cmdLineTests/openssl/playlist.xml#L66 instead?

Reply:
I agree. If it is not supported in FIPS140-2, then we should use `<feature>FIPS140_2:nonapplicable</feature>` for permanent excludes.

Reply:
Excluded it permanently.